### ISSUE TITLE
Why keras.applications.vgg19 not have dropout layers?

### ISSUE THREAD

**Issue by hawklucky** [keras-team/keras#7944]
Why keras.applications.vgg19 not have dropout layers?
If I fine-tune vgg19,  should I just add it ?

**Comment by harshini-gadige** [comment]
Hi

Since this is not a bug/feature request, request you to post these kind of questions in any of the following.

https://keras.io/
https://keras.io/#support
https://gitter.im/Keras-io/Lobby

Thanks!
